### PR TITLE
commenting out default servicemonitor auth to avoid checkov CKV_SECRET_6

### PR DESCRIPTION
**Pull request by rwd5213**
## what

Commenting out the default password and username setting for service monitor authentication to avoid running into the checkov SECRET_6 compliance check.

## why

Since its disabled by default any way there should be no harm in commenting out the actual setting of the password and prevents people from deploying with a default password if they do enable the auth check for the service monitor. Commented out and left for reference of users that want to set it.

## tests

Running checkov against charts with it commented out does not make the error occur.

### CODE REVIEW

File: charts/atlantis/values.yaml [hunk 1]
```diff
@@ -508,10 +508,10 @@ servicemonitor:
...
       # authentication based on an external secret
       enabled: false
-      name: atlantis-env
-      keys:
-        username: USERNAME
-        password: ATLANTIS_WEB_PASSWORD
+      # name: atlantis-env
+      # keys:
+        # username: USERNAME
+        # password: ATLANTIS_WEB_PASSWORD
```
Comment (GMartinez-Sisti) [inline charts/atlantis/values.yaml:514]: nit: keep the comment blocks aligned, similar to existing ones:
```suggestion
      # name: atlantis-env
      # keys:
      #   username: USERNAME
      #   password: ATLANTIS_WEB_PASSWORD
```

**Comment by GMartinez-Sisti** [comment]
Hi @rwd5213, even thought there's no actual change, it's a good practice to bump the chart. Bumping the patch version would be enough.

**Comment by rwd5213** [comment]
@GMartinez-Sisti Fixed both, thanks

**Review by GMartinez-Sisti — APPROVED** [review]
LGTM. I approve although I’m not a maintainer. I just help around 🙂

**Review by jamengual — APPROVED** [review]

**Pull request merged**